doc/conf.py

```diff
@@ -42,6 +42,10 @@
 
 # generate autosummary even if no references
 autosummary_generate = True
+autodoc_default_options = {'inherited-members': None}
+numpydoc_class_members_toctree = False
+numpydoc_attributes_as_param_list = True
+default_role = 'autolink'  # XXX silently allows bad syntax, someone should fix
```

Comment:
I am a bit confused by autodoc vs autosummary. Both seem to be sphinx extensions and you are adding some config values here.

But reading through the pages:

- [autosummary](https://www.sphinx-doc.org/en/master/usage/extensions/autosummary.html)
- [autodoc](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html)

I actually don't see how they work together (or not). Do you have an easy/short explanation?

Reply:
No, I'm not sure of how they interact, just have seen a lot of `doc/conf.py` files so had an idea of which config values could be used to fix linking problems (here they are taken from MNE's `conf.py`)

Reply:
okay, thanks anyways :-)